src/components/layout/Footer.less

```diff
@@ -53,6 +54,46 @@
   }
 }
 
+.footer__privacy-policy-area {
+  display: flex;
+  flex-basis: 20%;
+}
+
+.footer__privacy-policy {
+  vertical-align: middle;
+  display: inline-flex;
+  flex-direction: column;
+  flex-grow: 1;
+  flex-basis: 20%;
+  justify-content: center;
+  height: 100%;
+  color: white;
+  text-decoration: none;
+  &:hover {
```

Comment:
same here with BEM we try to avoid nesting

Reply:
also for this case, is there a way to get the `&:hover` functionality without nesting? seems like this pattern is used [elsewhere in the code](https://github.com/uc-cdis/data-portal/blob/6723c9f14061f81db57ee26e124567e0ddd29265/src/UserAgreement/Quiz.less#L74-L76) too

Reply:
I think you can write the same way as original css? Something like https://github.com/uc-cdis/data-portal/blob/63408352a83eb428827ee9f04254bb310a2c9fab/src/components/layout/NavBar.less#L97